voidcrew/code/controllers/subsystem/overmap.dm

```diff
@@ -265,17 +265,26 @@ SUBSYSTEM_DEF(overmap)
 		surface = null
 	vlevel.fill_in(surface, target_area)
 
+	if(mapgen)
+		mapgen.generate_terrain(vlevel.get_unreserved_block())
+
 	if(ruin_type)
 		var/turf/ruin_turf = locate(rand(
 			vlevel.low_x+6 + vlevel.reserved_margin,
 			vlevel.high_x-ruin_type.width-6 - vlevel.reserved_margin),
 			vlevel.high_y-ruin_type.height-6 - vlevel.reserved_margin,
 			vlevel.z_value
 			)
-		ruin_type.load(ruin_turf)
 
-	if(mapgen)
-		mapgen.generate_terrain(vlevel.get_unreserved_block())
+		var/list/ruin_bounds = block(locate(max(ruin_turf.x, 1),			max(ruin_turf.y, 1),			 ruin_turf.z),
```

Comment:
whats with the weird spacing here?

Reply:
i take it from /datum/map_template/proc/load(turf/T, centered = FALSE)